packages/hardhat/hardhat.config.ts

```diff
@@ -27,7 +29,7 @@ const ETHERSCAN_API_KEY = process.env.ETHERSCAN_API_KEY || "Your etherscan API k
 const PRIVATE_KEY = process.env.PRIVATE_KEY || "1dff2e1c343e5ca5787c71b0f557d9b2bafc420dccbc0724e723f48cdc0cdd74";
```

Comment:
@KovacZan clean this check lates PR on merged on contracts-phd repo... ... clean in a fresh PR please

Reply:
This is needed for `hh compile`
If we don't include this hardhat can't compile and tests can't run on gh actions.